.github/workflows/release_win_freethreading.yml

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: WindowsRelease (freethreading)
+
+on:  # Specifies the event triggering the workflow
+  workflow_call:  # Indicates that this is a reusable workflow
+    inputs:
+      os:
+        required: true
+        type: string
+        
+permissions:  # set top-level default permissions as security best practice
+  contents: read
+
+jobs:
+  build:
+    if: github.event_name != 'pull_request' || startsWith( github.base_ref, 'rel-') || contains( github.event.pull_request.labels.*.name, 'run release CIs')
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: ['3.13t']
+        architecture: ['x64']
+    steps:
+    - name: Checkout ONNX
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      with:
+         path: ./onnx
+         submodules: 'recursive'
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: Quansight-Labs/setup-python@v5
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: ${{ matrix.architecture }}
+
+    - name: Add msbuild to PATH
+      uses: microsoft/setup-msbuild@6fb02220983dee41ce7ae257b6f4d8f9bf5ed4ce # v2.0.0
+      with:
+        msbuild-architecture: ${{ matrix.architecture }}
+
+    - name: Install Python dependencies
+      run: |
+        python -m pip install -q --upgrade pip
+        cd onnx
+        python -m pip install -v -r requirements-dev_freethreading_win.txt
+        python -m pip install cmake
```

Comment:
Should we add cmake to requirements-dev_freethreading_win?

Reply:
I can try this. As long as we have / need so many requirements files it's definitely an option

Reply:
I changed that